### PR TITLE
Integerate bare gemm with new shell wrapper

### PR DESCRIPTION
**Pull request by xiaoling-yi**
In this RP, we 
* adopt the baregemm for new integration method
* add a new wrapper for the new integration interface

### CODE REVIEW

**Comment by rgantonio** [comment]
Do you need the PR review right now or you're still working on some fixes? Also when you make this PR, have the corresponding PR in SNAX cluster. So that we can see what changes need to be done in the SNAX cluster too. 😄

**Comment by xiaoling-yi** [comment]
Yes! Here is: https://github.com/KULeuven-MICAS/snitch_cluster/pull/114

Let's merge these changes to the main first. Then we can fix Bender in SNAX cluster points to the main.

File: Makefile [hunk 1]
```diff
@@ -1,18 +1,29 @@
 MK_DIR   := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
 
+define gen_sv_file
+	mkdir -p $(MK_DIR)generated/gemm && cd $(MK_DIR) && sbt "runMain gemm.$(1)"
```
Comment (rgantonio) [inline Makefile:4]: Just watch out for the `mill` update later 😄

**Review by rgantonio — APPROVED** [review]
Nice!

**Pull request merged**